pkg/cmd/run/view/view.go

```diff
@@ -212,13 +212,31 @@ func runView(opts *ViewOptions) error {
 		return fmt.Errorf("failed to get run: %w", err)
 	}
 
+	if opts.Exporter != nil {
```

Comment:
Move `Exporter` check here. It does not make sense to prompt job selection when using `--json`.